### DISCUSSION TITLE
Lock default branches

### DISCUSSION THREAD

**Discussion by flavioheleno** [beluga-php/docker-php#58] [General]
As I said in one of the issues we're discussing, I think it would be healthier now to lock the default branches and start working with PRs instead of commiting changes right on it.

Any objections to this?

cc @Rid @rodrigoaguilera

**Comment by rodrigoaguilera** [comment]
Please lock them. 

I will try to review open PRs but my approval only means that the code is alright to the best of my knowledge. I think it will take a bit more time for me to get familiar with the code to even merge a PR.

**Comment by Rid** [comment]
@flavioheleno happy for you to go ahead 👍

**Comment by flavioheleno** [comment]
Done!
